index.js

```diff
@@ -245,6 +245,12 @@ module.exports = function (app) {
           description: "When enabled the vessels position will be stored",
           default: false
         },
+        separateLatLon: {
+          type: "boolean",
+          title: "Latitude and Longitude as a separate measurements to Influxdb",
```

Comment:
Remove `a`, measurements is plural.

Reply:
Is the note in the readme enough?

Reply:
It's enough, but the wording is a bit odd and not very clear. I think you should use the expression _as separate measurements _ so that the doc is in line with the field description.